test/pom.xml

```diff
@@ -120,6 +124,16 @@
             <version>${hamcrest.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.spotbugs</groupId>
+            <artifactId>spotbugs-annotations</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.spotbugs</groupId>
+            <artifactId>spotbugs-annotations</artifactId>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Why are these added?
* You do not seem to add any annotation in test-container
* Why should it be twice there?
* Should it be optional if it is annotation only?

Reply:
This is redundancy thanks for the point. But one is correct I had issue with:
```
ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project test: Compilation failure: Compilation failure:
[ERROR] /Users/morsak/Documents/Work/strimzi-kafka-operator/test/src/main/java/io/strimzi/test/timemeasuring/TimeMeasuringSystem.java:[10,39] package edu.umd.cs.findbugs.annotations does not exist
[ERROR] /Users/morsak/Documents/Work/strimzi-kafka-operator/test/src/main/java/io/strimzi/test/timemeasuring/TimeMeasuringSystem.java:[91,6] cannot find symbol
[ERROR]   symbol:   class SuppressFBWarnings
[ERROR]   location: class io.strimzi.test.timemeasuring.TimeMeasuringSystem
[ERROR] -> [Help 1]
```

Reply:
Oh, sorry. I somehow thought it was the pom.xml in test-container. So ignore this.